### PR TITLE
Add BTM agent instructions to QuickStart

### PR DESCRIPTION
**Pull request by dometec**

### CODE REVIEW

File: src/main/jbake/content/docs/user/quick-start.adoc [hunk 1]
```diff
@@ -156,6 +156,13 @@ TIP: Optional: A single Hawkular WildFly Agent instance can also manage multiple
 Now you can start the managed WildFly instance, keeping in mind that if you are following this QuickStart your Hawkular Server is probably already running on port 8080, so you need to run this instance on a different port by starting the server with `sh bin/standalone.sh -Djboss.socket.binding.port-offset=100` for instance.
 Now if you have correcty set up the agent, it will report into your Hawkular Server at http://localhost:8080/hawkular-ui/app/app-list.
 
+To push data about Application Performance Management / Business Transaction Management you need to install the (second) relative agent available in `btm` folder. Open the `btm\setenv.sh` file and set the correct URL user/pass and do:
+----
+source btm\setenv.sh
```
Comment (jkremser) [inline src/main/jbake/content/docs/user/quick-start.adoc]: would you mind changing the backslash to normal slash? We are not windows haters, it's just, it's not supported on windows, afaik :]

File: src/main/jbake/content/docs/user/quick-start.adoc [hunk 2]
```diff
@@ -156,6 +156,13 @@ TIP: Optional: A single Hawkular WildFly Agent instance can also manage multiple
 Now you can start the managed WildFly instance, keeping in mind that if you are following this QuickStart your Hawkular Server is probably already running on port 8080, so you need to run this instance on a different port by starting the server with `sh bin/standalone.sh -Djboss.socket.binding.port-offset=100` for instance.
 Now if you have correcty set up the agent, it will report into your Hawkular Server at http://localhost:8080/hawkular-ui/app/app-list.
 
+To push data about Application Performance Management / Business Transaction Management you need to install the (second) relative agent available in `btm` folder. Open the `btm\setenv.sh` file and set the correct URL user/pass and do:
```
Comment (jkremser) [inline src/main/jbake/content/docs/user/quick-start.adoc]: ..and here is the second

**Comment by pilhuhn** [comment]
@objectiser Would you mind to review as well?

**Comment by dometec** [comment]
Il giorno ven, 08/04/2016 alle 10.58 -0700, Heiko W. Rupp ha scritto:

> @objectiser Would you mind to review as well?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub

Sorry for the delay.
Done.

Thank you,
Domenico

**Comment by objectiser** [comment]
Thanks Domenico

**Pull request merged**